modules/web/js/ballerina/diagram2/views/default/components/nodes/top-level-nodes.jsx

```diff
@@ -95,6 +95,10 @@ class TopLevelNodes extends React.Component {
             this.props.model.viewState.packageDefExpanded = false;
         }
         if (this.packageDefValue) {
+            // If there is a semi-colon at the end of the package name, remove it
+            if (this.packageDefValue.slice(-1) === ';') {
```

Comment:
You can use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/endsWith function here.

Reply:
Used the _trimEnd function of Lodash  https://lodash.com/docs/4.17.4#trimEnd 
Fixed with commit 05f4d18